### ISSUE TITLE
Error on startup: Argument for @NotNull parameter 'language' 

### ISSUE THREAD

**Issue by nextminds** [styled-components/webstorm-styled-components#9]
I've just installed v1.0-alpha on Webstorm 2017.2.3 and I get the following error on startup:

```
Argument for @NotNull parameter 'language' of com/intellij/psi/impl/source/tree/injected/MultiHostRegistrarImpl.startInjecting must not be null
```

Using:
WebStorm 2017.2.3
Build #WS-172.3968.27, built on August 29, 2017
JRE: 1.8.0_152-release-915-b11 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

Full stack trace:
```
Argument for @NotNull parameter 'language' of com/intellij/psi/impl/source/tree/injected/MultiHostRegistrarImpl.startInjecting must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'language' of com/intellij/psi/impl/source/tree/injected/MultiHostRegistrarImpl.startInjecting must not be null
	at com.intellij.psi.impl.source.tree.injected.MultiHostRegistrarImpl.startInjecting(MultiHostRegistrarImpl.java)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl$PsiManagerRegisteredInjectorsAdapter.lambda$getLanguagesToInject$0(InjectedLanguageManagerImpl.java:490)
	at SCInjector.getLanguagesToInject(SCInjector.java:43)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl$PsiManagerRegisteredInjectorsAdapter.getLanguagesToInject(InjectedLanguageManagerImpl.java:494)
	at com.intellij.psi.impl.source.tree.injected.InjectedPsiCachedValueProvider$MyInjProcessor.process(InjectedPsiCachedValueProvider.java:82)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.processInPlaceInjectorsFor(InjectedLanguageManagerImpl.java:461)
	at com.intellij.psi.impl.source.tree.injected.InjectedPsiCachedValueProvider.doCompute(InjectedPsiCachedValueProvider.java:62)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.probeElementsUp(InjectedLanguageUtil.java:312)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.enumerate(InjectedLanguageUtil.java:144)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:241)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:203)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.a(LineMarkersPass.java:111)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:96)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:106)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:438)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1160)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:431)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:548)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:493)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:430)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:406)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:142)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:249)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:404)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:165)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)


java.lang.IllegalArgumentException: Argument for @NotNull parameter 'language' of com/intellij/psi/impl/source/tree/injected/MultiHostRegistrarImpl.startInjecting must not be null
	at com.intellij.psi.impl.source.tree.injected.MultiHostRegistrarImpl.startInjecting(MultiHostRegistrarImpl.java)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl$PsiManagerRegisteredInjectorsAdapter.lambda$getLanguagesToInject$0(InjectedLanguageManagerImpl.java:490)
	at SCInjector.getLanguagesToInject(SCInjector.java:43)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl$PsiManagerRegisteredInjectorsAdapter.getLanguagesToInject(InjectedLanguageManagerImpl.java:494)
	at com.intellij.psi.impl.source.tree.injected.InjectedPsiCachedValueProvider$MyInjProcessor.process(InjectedPsiCachedValueProvider.java:82)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.processInPlaceInjectorsFor(InjectedLanguageManagerImpl.java:461)
	at com.intellij.psi.impl.source.tree.injected.InjectedPsiCachedValueProvider.doCompute(InjectedPsiCachedValueProvider.java:62)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.probeElementsUp(InjectedLanguageUtil.java:312)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.enumerate(InjectedLanguageUtil.java:144)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.findInside(InjectedLanguageUtil.java:409)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.tryOffset(InjectedLanguageUtil.java:388)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.findInjectedElementNoCommit(InjectedLanguageUtil.java:368)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.findInjectedPsiNoCommit(InjectedLanguageUtil.java:271)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.getEditorForInjectedLanguageNoCommit(InjectedLanguageUtil.java:179)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorPsiDataProvider.getData(TextEditorPsiDataProvider.java:62)
	at com.intellij.openapi.fileEditor.ex.FileEditorManagerEx.getData(FileEditorManagerEx.java:180)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorComponent.getData(TextEditorComponent.java:268)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl$PsiAwareTextEditorComponent.getData(PsiAwareTextEditorImpl.java:118)
	at com.intellij.ide.impl.DataManagerImpl.a(DataManagerImpl.java:86)
	at com.intellij.ide.impl.DataManagerImpl.a(DataManagerImpl.java:73)
	at com.intellij.ide.impl.DataManagerImpl.access$300(DataManagerImpl.java:58)
	at com.intellij.ide.impl.DataManagerImpl$MyDataContext.a(DataManagerImpl.java:400)
	at com.intellij.ide.impl.DataManagerImpl$MyDataContext.getData(DataManagerImpl.java:377)
	at com.intellij.openapi.actionSystem.DataContextWrapper.getData(DataContextWrapper.java:38)
	at com.intellij.openapi.actionSystem.AnActionEvent$1.getData(AnActionEvent.java:145)
	at com.intellij.openapi.actionSystem.DataKey.getData(DataKey.java:69)
	at com.intellij.openapi.editor.actionSystem.EditorAction.getEditor(EditorAction.java:91)
	at com.intellij.openapi.editor.actionSystem.EditorAction.update(EditorAction.java:127)
	at com.intellij.openapi.actionSystem.AnAction.beforeActionPerformedUpdate(AnAction.java:240)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:129)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:599)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.a(IdeKeyEventDispatcher.java:479)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:213)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:621)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by daedlock** [comment]
@nextminds that should be fixed in the next release. Until then, you are welcome to use this jar https://github.com/styled-components/webstorm-styled-components/blob/c1942612e2385e313a483f76e93a56597f93789a/webstorm-styled-components.jar?raw=true

**Comment by daedlock** [comment]
@nextminds an update!

We just released `v1.0` with a possible fix for this problem. Make sure to use `webstorm-styled-components.zip`

https://github.com/styled-components/webstorm-styled-components/releases/tag/v1.0

**Comment by daedlock** [comment]
Please install using the IDE plugin manager.
